test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -253,3 +235,113 @@ func schedulePods(config *testConfig) int32 {
 		time.Sleep(1 * time.Second)
 	}
 }
+
+// mutateNode returns the strategy needed for creation of nodes.
+// TODO: It should take the nodespec and return the modified version of it.
+func (na nodeAffinity) mutateNode(numNodes int) []testutils.CountToStrategy {
```

Comment:
add comment about backward compatibility, i.e. "heres why we're not really mutating "